components/notification-sender-config/org.wso2.carbon.identity.notification.sender.tenant.config/src/test/resources/testng.xml

```diff
@@ -0,0 +1,29 @@
+<!--
+  ~ Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
+  ~
+  ~ WSO2 LLC. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="WebSubHubEventAdapterTestSuite">
+
+    <test name="NotificationSenderTenantConfigTest" preserve-order="true" parallel="false">
+        <classes>
+            <class name="org.wso2.carbon.identity.notification.sender.tenant.config.NotificationSenderManagementServiceImpl"/>
```

Comment:
these should be the test classes.

Reply:
resolved in https://github.com/wso2-extensions/identity-event-handler-notification/pull/179/commits/8d4061dc9217f85e6940d3e966fcd4cb856ad596